SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -272,8 +272,9 @@ - (void)sendData:(NSData *)data {
         NSOutputStream *ostream = self.session.easession.outputStream;
         NSMutableData *remainder = data.mutableCopy;
 
-        while (remainder.length != 0) {
-            if (ostream.streamStatus == NSStreamStatusOpen && ostream.hasSpaceAvailable) {
+        while (ostream.streamStatus == NSStreamStatusOpen &&
+               remainder.length != 0) {
```

Comment:
Style-wise, unless there's many checks, we generally prefer to leave two checks on the same line and let wrapping occur naturally if necessary.